### ISSUE TITLE
port use error

### ISSUE THREAD

**Issue by alh321** [nextstrain/auspice#1309]
I've been getting the error ` [error]	Port 4000 is currently in use by another program`. When trying to run auspice view. I installed through conda as recommended (though with nodejs=11.14.0 - was having some trouble getting v12), tried to run with the test data as recommended, and got this error. I see that this was noted as an issue that appeared when nextstrain view was also running (and seems to have been limited to macs; https://github.com/nextstrain/cli/issues/48), but I am running on Linux.

**Comment by jameshadfield** [comment]
Hey @alh321 -- It's hard to know what program is using that port without more info, however you can try `lsof -n -i :4000 |  grep LISTEN`and see if the output is helpful. For a quick fix, if you run `PORT=4001 auspice view` then auspice should run on 4001 instead.

**Comment by alh321** [comment]
Ah apologies! I should have mentioned that I tried `lsof -n -i :4000 | grep LISTEN` after `auspice view` terminated with that error, and it returned nothing. Similarly, I installed auspice on another machine and tried `lsof -n -i :4000 | grep LISTEN` before running it, and it also returned nothing, but subsequently running `auspice view` still resulted that in that port error, so I assumed that `auspice view` was trying to use the same port for multiple processes.
